frontend/WebUI-diff/WebUI.diff

```diff
@@ -0,0 +1,234 @@
+diff --color -rc webconsole-orig/backend/WebUI/api_webui.go webconsole/backend/WebUI/api_webui.go
```

Comment:
Hi @kishiguro 
Do we need this patch file after PR merged?

Reply:
@ianchen0119 No, no need of merging the patch.  it is included in the PR.

Reply:
@ianchen0119 diff has been removed to avoid confusion.